package.json

```diff
@@ -6,9 +6,7 @@
     "build": "webpack --mode production",
     "buildProd": "webpack --mode production",
     "dev": "webpack --mode development --progress --watch",
-    "test": "jest",
-    "storybook": "start-storybook -p 6006 -s public",
-    "build-storybook": "build-storybook -s public"
```

Comment:
Why are `storybook` and `build-storybook` deleted in this PR?

Reply:
good point. I think that could be for #8

Reply:
yes , this version do not have storybook , so i delete storybook config